### PR TITLE
Create Edit Team Member View

### PR DESCRIPTION
**Pull request by adamsmithcentrica**
- Refactored models to conform with ember relational structure
- Added in edit view with 404 error handling
- Included acceptance tests

### CODE REVIEW

File: tests/unit/models/member-test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
```
Comment (afreeman123) [inline tests/unit/models/member-test.js]: We can test the relationships of hasMany I think - https://guides.emberjs.com/release/testing/testing-models/

**Comment by adamstacey7** [comment]
i thought we agreed to not do the JSON structure that way? (`mirage/config.js`)

If its easier for learning then can leave it, but the other way with JSON obj with `key:value` for the members would be better && more scaleable.

...but for the purpose of learning can leave it that way if its easier for the time being

**Comment by adamsmithcentrica** [comment]
@adamstacey7 That's the way ember data wants it 'out the box'/how the tutorial structures it. We could look to refactor it at a later stage as a learning exorcise so we know how to do both?

**Comment by afreeman123** [comment]
#20

**Pull request merged**